src/core/helpers.jl

```diff
@@ -0,0 +1,33 @@
+nw_ids(pm::_IM.AbstractInfrastructureModel) = _IM.nw_ids(pm, _pm_it_sym)
```

Comment:
For all functions in this file I am wondering why the type here is not `AbstractPowerModel`?  One possible advantage is the versions of these functions can co-exist in the same name space and have the correct semantics based on the argument type they are given.

Reply:
Ah, I see your point. I need to modify some downstream functionality in GasPowerModels, then I will change back the parameter type of these functions.